alpine-nix-rails-nginx/Dockerfile

```diff
@@ -0,0 +1,53 @@
+# dkubb/alpine-nix-rails-nginx
+
+FROM dkubb/alpine-nix:latest
+MAINTAINER Dan Kubb <dkubb@fastmail.com>
+
+EXPOSE 80
+
+# Configure nix
+COPY config.nix /root/.nixpkgs/config.nix
+
+# Install system dependencies
+RUN nix-env --install --fallback \
+    acl-2.2.52 \
+    bundler-1.10.5 \
+    gcc-wrapper-5.2.0 \
+    git-2.5.0 \
+    gnumake-4.1 \
+    libressl-2.2.2 \
+    libxml2-2.9.2 \
+    libxslt-1.1.28 \
+    nginx-1.8.0 \
+    nodejs-0.12.7 \
+    ruby-2.2.2-p0 \
```

Comment:
Should we think about using our own package shipping `2.3.3` here? `ruby-2.2.2-p0` was as I remember totally broken and segfaulting a lot. Only more recent patchlevels fixed it.